packages/volto-slate/news/5518.bugfix

```diff
@@ -0,0 +1 @@
+When copying a text block—implemented in Slate—using the sidebar that contains UUIDs attached to elements in the data, the UUIDs get copied into the new block, making them not unique. Now instead of duplicating the UUIDs, a unique nanoid will be generated and assigned to the copy, preserving uniqueness. @dobr1408
```

Comment:
Not sure what the sidebar bit is about.

```suggestion
When copying a text block—implemented in Slate—that contains UUIDs attached to elements in the data, the UUIDs get copied into the new block, making them not unique. Now instead of duplicating the UUIDs, a new UUID will be generated and assigned to the copy, preserving uniqueness. @dobr1408
```

Reply:
"sidebar"  is there to emphasies that is about copying using volto, not the slate default copying method

Reply:
I think that part of the volto UI is called the toolbar, not the sidebar

Reply:
Given that we are "copying a text block", I assume the context is obvious that it is a Volto block, not a Slate element, that is being copied, so we don't need to name whatever that thing is called. 😁 I agree with @davisagli's suggestion here.

Side note, after a quick search in the docs, the thingie on the right-hand side in Volto is usually called "sidebar". The thingie on the left-hand side is usually called "Plone toolbar" or "toolbar", and the thingie in a text editor is called just "toolbar" and sometimes "editor toolbar" or "Slate toolbar". It would be good to officially name these thingies. Great, I just made more documentation work for me. 🙄